src/support_sphere/test/unit_tests/auth_service_test.dart

```diff
@@ -17,14 +17,10 @@ void main() {
   // Refer: https://docs.flutter.dev/cookbook/testing/unit/introduction#5-combine-multiple-tests-in-a-group
 
   group('AuthService SignUp Code Validation Tests', () {
-    test('isSignupCodeValid returns true for valid code', () async {
-      final isValid = await authService.isSignupCodeValid('SUPPORT');
-      expect(isValid, isTrue);
-    });
-
-    test('isSignupCodeValid returns false for invalid code', () async {
-      final isValid = await authService.isSignupCodeValid('INVALID');
-      expect(isValid, isFalse);
-    });
+    // TODO: Ignore test.. need to have supabase instance running
```

Comment:
should we link to an issue that would resolve this todo if said issue were completed?